SU-HSDO/suhumsci
# Release: FourKitchens Sprint 8

## Summary
- SHS-77 : Mega Menu unpublished link styles
- SHS-29: add title field to timeline component and style
- SHS-81: bug fix for traditional layered slider at large screen size

## Need Review By (Date)
9/8

## Urgency
low

## Steps to Test
SHS-29: For HS-Colorful or HS-Traditional
- Import the config and clear the cache if need be
- Go to the example page for vertical timelines here: components/vertical-timeline-without-and-collection
- Edit that page and add titles to a few of the examples
- The title should be an H2 in markup, but styles differently
- Make sure it shows up in the traditional theme

SHS-77: For HS-Colorful or HS-Traditional
- Enable the new megamenu, go to the main menu (/admin/structure/menu/manage/main) and unpublish a page.
- Verify that it gets the golden background to signify its unpublished like in the screenshots below.

SHS-81 - For HS-Tradition site:
- Create a new page. In the main region add a collection of two. Both of these items should be the Hero Layered Slider option.
- Make sure the theme is Traditional.
- View the page at greater than 1500px. Notice that the arrows for the slider are not off the box.
- Check to make sure this doesn't break a single layered hero slider (non collection)
- Check to make sure this doesn't affect the colorful theme (this should only affect traditional - there's a separate bug for colorful).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
